scripts/pi-hole/js/messages.js

```diff
@@ -178,19 +166,21 @@ function delMsg(ids) {
         utils.showAlert(
           "success",
           "far fa-trash-alt",
-          "Successfully deleted " + ids.length + " message" + (ids.length > 1 ? "s" : ""),
-          ""
+          "Successfully deleted message",
```

Comment:
The ID is returned on error. I think we could also show the ID on success.

```suggestion
          "Successfully deleted message ID:" + id,
```

Without the ID, if you delete 5 messages there will be 5 toasts with the exactly same text.

Reply:
Two comments: If we use your suggestion, I'd suggest an additional space between `:` and `"`. Otherwise, how about putting the ID into the second line (currently `null` to be disabled)? This is also where we are putting domains/regex, etc. in other toasts.

Reply:
I'm OK with both solutions:
- space after "ID:" (suggestion updated) or
- replacing the `null` parameter with `ID: XX`.

I just don't want to remove the string "ID:" to avoid confusion between the ID and the count of deleted items.